backend/src/main/java/com/example/dontwastetomuch/game/GameService.java

```diff
@@ -35,6 +36,7 @@ public Game getOneGame(String gameId) {
         return gameRepo.findById(gameId).orElseThrow();
     }
 
+
     public void editGame(Game game) {
```

Comment:
here you could get the User from a principal in the controller and check if it has the role Admin. 
so you could Approve games edited by Admins.

Reply:
yeah, i did not touch the rols at the moment, but i think it would be the right place to do it 👍